src/util/config/global-path.ts

```diff
@@ -17,7 +38,16 @@ const getNowDir = () => {
   const customPath = args['global-config'];
 
   if (!customPath) {
-    return path.join(homedir(), '.now');
+    const nowDir = path.join(
+      process.env.XDG_DATA_HOME || path.join(homedir(), '.local', 'share'),
+      'now'
+    );
+
+    if (oldConfigExists(oldConfigPath)) {
+      mvOldConfig(oldConfigPath, nowDir);
```

Comment:
Missing `await` here. But aside from that I think we don't want to move the old config if it exists, but rather just use it as-is.

Reply:
It's removed :)

Reply:
And I simply use the old config now.